src/Migrator.EF6.Tools/Migrator.EF6.Tools.csproj

```diff
@@ -0,0 +1,51 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>A .NetCore CLI tool to enable EF6 migrations in an Asp.Net Core app.</Description>
+    <VersionPrefix>2.0.2</VersionPrefix>
+    <Authors>Mohammad Rahhal</Authors>
+    <TargetFrameworks>net462;netcoreapp1.0</TargetFrameworks>
+    <RuntimeIdentifiers>win7-x64;win7-x86;osx.10.10-x64;osx.10.11-x64;ubuntu.14.04-x64;ubuntu.16.04-x64;centos.7-x64;rhel.7.2-x64;debian.8-x64;fedora.23-x64;opensuse.13.2-x64</RuntimeIdentifiers>
+    <RuntimeIdentifier Condition=" '$(TargetFramework)' == 'net462' ">win7-x86</RuntimeIdentifier>
+    <AssemblyName>dotnet-ef</AssemblyName>
+    <OutputType>Exe</OutputType>
+    <PackageId>Migrator.EF6.Tools</PackageId>
+    <PackageTags>aspnetcore;aspnet5;ef6;migrations;dotnet</PackageTags>
+    <PackageProjectUrl>https://github.com/mrahhal/Migrator.EF6</PackageProjectUrl>
+    <PackageLicenseUrl>https://github.com/mrahhal/Migrator.EF6/blob/master/LICENSE.txt</PackageLicenseUrl>
+    <PackageTargetFallback Condition=" '$(TargetFramework)' == 'netcoreapp1.0' ">$(PackageTargetFallback);dnxcore50</PackageTargetFallback>
+    <RuntimeFrameworkVersion Condition=" '$(TargetFramework)' == 'netcoreapp1.0' ">1.1.0</RuntimeFrameworkVersion>
+    <Version>2.0.4</Version>
```

Comment:
Ironic after all the talk about the "auto migration path", because the tool seemed to make some pretty obvious mistakes.

Alright, I'll give this another pass soon. And now that I think about it, the stable tooling RTMs tomorrow (March 7th) with VS15, so maybe we should wait for that and do last edits before merging?